### DISCUSSION TITLE
[2021-12-19] 주간 회의록

### DISCUSSION THREAD

**Discussion by HyejinYang** [team-mokcha/corabal-beta#53] [주간 회의록]
## 회의 내용
**1. 디자인 추가 예정**
  - 오늘 0잔 기록' 완료 시의 Today 컴포넌트 디자인 개선
  - 전역에서의 Loading spinner 컴포넌트 추가

**2. 컴포넌트 / 로직 / 전역 상태 추가**
  - disableButton 컴포넌트 추가
  - '오늘 기록 마감' 상태로 돌입할 경우, 플로팅 버튼으로 더 이상 커피 기록을 추가할 수 없고 관련 알림을 띄워주는 로직 추가
  - 기록 추가 or 마감에 따른 커피 로그 전역 상태 관리 필요할 것으로 보임
 
**3. DB 필드 추가**
  - normal_cup_record에 각 기록별로 uuid 추가(uuid 생성은 라이브러리 이용)

**Comment by HyejinYang** [comment]
@alveloper 오늘 얘기한 내용 주간으로 퉁쳐서 일요일 회의록에 적었습니다

논의 배경은 해당 이슈 참고 - #14 [FEAT] 커피 로그 생성 기능 구현

**Comment by ghost** [comment]
오후 회의 이후 정리된 룰
1. ~2주 동안 이슈 내의 서브 이슈(?)를 해결하기
2. 수연: 프로필 건들기
3. 혜진: 메인 내에 있는 로직 분리 시도해보기
